src/middleware/service.availability.middleware.ts

```diff
@@ -27,14 +24,5 @@ export const serviceAvailabilityMiddleware = (req: Request, res: Response, next:
     return res.render(SERVICE_OFFLINE_PAGE);
   }
 
-  if (
-    (req.path === STARTING_NEW_URL || req.path.endsWith(`/${RESUME}`))
-    &&
-    !isActiveFeature(FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
-    logger.infoRequest(req, "Save and resume flag is disabled - displaying service offline page");
-    return res.render(SERVICE_OFFLINE_PAGE);
-  }
-
-  // feature flag SHOW_SERVICE_OFFLINE_PAGE is false - carry on as normal
```

Comment:
Strictly speaking, the comment on line 38 shouldn't be removed, although as it is not entirely accurate, this is probably fine. Alternatively, we reinstate it and correct it to be something like 'Service not offline, carry on as normal'. Not sure it adds much...

Reply:
Reintroduced and added detail to the comment